cves/kernel/CVE-2019-13648.yml

```diff
@@ -405,8 +434,8 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: The system currently allows the user to modify the MSR register. Given the potential for misuse, access to this register should be abstracted to disallow direct interaction.
```

Comment:
I might add an explicit note about distrusting input. Like adding "thereby enforcing input distrust." at the end.

Also wrap to multiple lines.

Reply:
added